Created the method "usersInGroup" in the src\Authorization\FlatAuthorization.php file (#391)
lonnieezell/myth-auth: docs/authorization.md

	$authorize->removeUserFromGroup($userId, $group_id);
	$authorize->removeUserFromGroup($userId, 'moderators');

### usersInGroup()

> Lists all users in a single group. The group must already exist. The only parameter is the group you want to list. Both
> parameters can be ID or name. The data return is an array.

Reviewer: ```suggestion
Lists all users in a single group. The group must already exist. The only parameter is the group you want to list, which can be its ID or name. The data return is an array of user array rows. 
```
